.github/workflows/build_and_test.yml

```diff
@@ -16,13 +18,24 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Clone dependency packages
-      run: mkdir dependency_ws && vcs import dependency_ws < build_depends.repos
+      run: |
+        mkdir dependency_ws
+        vcs import dependency_ws < build_depends.repos
 
     - name: Install missing dependencies
-      run: sudo apt update && rosdep update && DEBIAN_FRONTEND=noninteractive rosdep install --from-paths . --ignore-src --rosdistro foxy -y
+      run: |
```

Comment:
@esteve what does `|` in the beginning mean? Is the newline here intentional?

Reply:
It's the syntax that Github actions use for multiline scripts, using `&&` for two commands is fine, but as soon the script grows, I prefer using multiple lines.